rosdep/python.yaml

```diff
@@ -154,6 +154,19 @@ gunicorn:
   fedora: [python-gunicorn]
   gentoo: [www-servers/gunicorn]
   ubuntu: [gunicorn]
+ifcfg-pip:
```

Comment:
Since we're using this for ROS 2 we'll want the key name for the platform installer to be `python3-ifcfg`. I think this key name is fine for pip. An alternative might be `python3-ifcfg-pip` but since the pip key is not python3 specific I don't feel a preference for it.